drake/systems/Simulation.h

```diff
@@ -103,8 +103,9 @@ inline bool handle_realtime_factor(const TimePoint& wall_clock_start_time,
 template <typename System>
 void simulate(const System& sys, double ti, double tf,
               const typename System::template StateVector<double>& xi,
-              SimulationOptions& options) {
-  if (options.realtime_factor < 0.0) options.realtime_factor = 0.0;
+              const SimulationOptions& options) {
+  SimulationOptions my_options = options;
+  if (my_options.realtime_factor < 0.0) my_options.realtime_factor = 0.0;
```

Comment:
FWIW, I did the grep for this, and didn't see any code that touched realtime_factor after simulate() or runLCM() is called.  (I did find that runLCM re-implements the "negative means 1.0" rule, but _shrug_.)